_pytest/_code/source.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import generators
+from __future__ import absolute_import, division, generators, print_function
```

Comment:
@nicoddemus in fact - the place where its used suggests incredibly bad api because any external user would pass wrong flags unless being told about the behavior of the default